src/nft/components/details/detailsV2/TabbedComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import Row from 'components/Row'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { containerStyles } from './shared'
+
+const TabbedComponentContainer = styled.div`
+  height: 288px;
+
+  ${containerStyles}
+`
+
+const TabsRow = styled(Row)`
+  gap: 32px;
+  margin-bottom: 12px;
+  width: 100;
+`
+
+const Tab = styled(ThemedText.SubHeader)<{ isActive: boolean; numTabs: number }>`
+  color: ${({ theme, isActive }) => (isActive ? theme.textPrimary : theme.textTertiary)};
+  line-height: 24px;
+  cursor: ${({ numTabs }) => (numTabs > 1 ? 'pointer' : 'default')};
+
+  &:hover {
+    opacity: ${({ numTabs, theme }) => numTabs > 1 && theme.opacity.hover}};
+  }
+`
+
+export interface Tab {
+  title: React.ReactNode
+  key: string
```

Comment:
it looks like you only use this as a unique identifier for rendering, so you could nuke this and just use array length and pass index in your map function. however i don't think it's necessarily bad to have this as like a "url slug" in case you want to like change the default tab to render somehow. but then i might rename it to `slug` or something

Reply:
Again I think it's a better object oriented pattern to control state based on a unique identifier for the element than it's array index.

Reply:
Yeah, agreed that unique identifier might be better as with keys we may have some unexpected behavior should we hide tabs dynamically in the future.